bundle.yaml

```diff
@@ -16971,8 +16971,8 @@ spec:
         - --kubelet-service=kube-system/kubelet
         - --logtostderr=true
         - --config-reloader-image=jimmidyson/configmap-reload:v0.4.0
-        - --prometheus-config-reloader=quay.io/coreos/prometheus-config-reloader:v0.41.0
-        image: quay.io/coreos/prometheus-operator:v0.41.0
+        - --prometheus-config-reloader=quay.io/prometheus-operator/prometheus-config-reloader:v0.41.0
+        image: quay.io/prometheus-operator/prometheus-operator:v0.41.0
```

Comment:
Are these images live already? Otherwise this might be a problem for anyone who wants to try this out.

Reply:
Yes, as commented in https://github.com/prometheus-operator/prometheus-operator/pull/3384#issuecomment-669151014

Reply:
Plus it is confirmed to be working: https://github.com/thaum-xyz/ankhmorpork/commit/88f461440745f294e62b2c902755a2a4a47a4d48 :)